common/src/transport_api/mod.rs

```diff
@@ -281,6 +287,26 @@ impl ReplyError {
             extra,
         }
     }
+
+    /// For deadline exceeded errors.
+    pub fn deadline_exceeded(resource: ResourceKind, source: String, extra: String) -> Self {
```

Comment:
Where are we using this?

Reply:
I had used it initially. Kept it in the PR because it can be used when there is appropriate scenario. We have DeaflineExceeded in ReplyErrorKind already.